app/en/home/build-tools/create-a-tool-with-secrets/page.mdx

```diff
@@ -219,24 +244,33 @@ if __name__ == "__main__":
 ### Run your MCP server
 
 <Tabs
-  items={["HTTP transport (default)", "stdio transport (for Claude Desktop)"]}
+  items={["stdio transport (default)", "http transport"]}
   storageKey="preferredTransport"
 >
   <Tabs.Tab>
 
 ```bash
-uv run secrets.py http
+uv run secrets.py stdio
 ```
 
-For HTTP transport, view your server's API docs at [http://127.0.0.1:8000/docs](http://127.0.0.1:8000/docs).
-
 </Tabs.Tab>
 <Tabs.Tab>
 
 ```bash
-uv run secrets.py stdio
+uv run secrets.py http
 ```
 
+For HTTP transport, view your server's API docs at [http://127.0.0.1:8000/docs](http://127.0.0.1:8000/docs).
+
+<Callout type="info">
+  For security reasons, Local HTTP servers do not currently support managed
+  authorization and secrets. If you need to use authorization or secrets, you
+  should use the stdio transport and configure the Arcade API key and secrets in
+  your MCP connection settings. If you intend to expose your HTTP MCP server to
```

Comment:
I think it would make more sense to just tell them to `arcade login` instead of referring to a general 'MCP connection settings'

Reply:
or define what `MCP connection settings` are somewhere